docs/src/modules/components/MarkdownDocs.js

```diff
@@ -205,6 +356,35 @@ function MarkdownDocs(props) {
                     ) : (
                       <div />
                     )}
+                    <NoSsr>
+                      <div>
+                        <Typography
+                          variant="subtitle1"
+                          component="span"
+                          className={classes.feedbackMessage}
+                        >
+                          {t('ratingMesssage')}
+                        </Typography>
+                        <Tooltip title={t('ratingYes')}>
```

Comment:
Any reason why the visual laber is different from the accessible label? Or is this actually intended as a description in which case you want to add `describeChild` (see https://github.com/mui-org/material-ui/pull/22729)

Reply:
Yep, I forgot to remove them when I added the tooltip. 🙂